prettyprinter/bench/LargeOutput.hs

```diff
@@ -1,5 +1,7 @@
-{-# LANGUAGE DeriveGeneric     #-}
-{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE DeriveGeneric        #-}
+{-# LANGUAGE FlexibleInstances    #-}
+{-# LANGUAGE OverloadedStrings    #-}
+{-# LANGUAGE TypeSynonymInstances #-}
```

Comment:
Is this change still necessary? I don’t see a change that would require those exts in this file.

Reply:
I've removed `FlexibleInstances` and `TypeSynonymInstances`.  Hopefully it still works.  Please approve workflows.